tests/cpp/test_tutorial.cpp

```diff
@@ -697,4 +697,201 @@ TEST_F(Tutorial, IdModelReshapeAnalysis) {
   }
 }
 
+TEST_F(Tutorial, PointwiseBroadcastTMA) {
+  CompileParams index32bit{DataType::Int32, 255, false};
+
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  constexpr at::ScalarType dtype = at::ScalarType::Float;
+
+  auto tv0 = makeContigTensor(3, aten_to_data_type(dtype));
+  auto tv1 = makeContigTensor(4, aten_to_data_type(dtype));
```

Comment:
To be consistent with https://github.com/NVIDIA/Fuser/pull/2130/files#diff-d21d811df5193bfc259b20940ca4e5fe43c6648a136c201f7884dbce4decbe2dR448-R449, can we create tensor with contiguity
```
T0[I1, I2, I3] contiguity: T, T, T
T1[I0, I1, I2, I3] contiguity: T, F, T, T
```